test/checks/color/color-contrast.js

```diff
@@ -880,7 +890,7 @@ describe('color-contrast', function() {
       assert.equal(checkContext._data.messageKey, 'shadowOnBgColor');
     });
 
-    it('fails if thick text shadows don\'t have sufficient contrast', function() {
+    it("fails if thick text shadows don't have sufficient contrast", function() {
```

Comment:
This should remain single quotes

Reply:
Prettier should be in charge of this.

Reply:
I did not make this change, some robot must have made the change

Reply:
How interesting. I hadn't seen prettier trigger off of the use of an escaped quote changing the string from single to double quotes. Wonder if it's a new thing in prettier as this hasn't happened before when we've run prettier over the whole codebase.